src/main/scala/com/microsoft/ml/spark/lightgbm/booster/LightGBMBooster.scala

```diff
@@ -1,12 +1,15 @@
 // Copyright (C) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in project root for information.
 
-package com.microsoft.ml.spark.lightgbm
+package com.microsoft.ml.spark.lightgbm.booster
```

Comment:
why the namespace change?

Reply:
moving params, booster and dataset into their own folders, because there are way too many files in lightgbm folder